Fix invalid html in i18n strings

### Problem

There are a few example of strings with incorrect HTML:

```
FAILED openlibrary/i18n/test_po_files.py::test_html_format[es-Continue</a>.] - xml.etree.ElementTree.ParseError: mismatched tag: line 1, column 59
FAILED openlibrary/i18n/test_po_files.py::test_html_format[es-<strong>Download the DAISY.zip</strong></a> for <a href="%(url)s">%(title)s</a>] - xml.etree.ElementTree.ParseError: mismatched tag: line 1, column 90
FAILED openlibrary/i18n/test_po_files.py::test_html_format[es-For example: <i><a href="/subjects/cheese">cheese</a>, <a href="/subjects/roman_empire">Roman Empire</a>, <a href="/subjects/psychology">psychology</a>] - xml.etree.ElementTree.ParseError: mismatched tag: line 1, column 202
FAILED openlibrary/i18n/test_po_files.py::test_html_format[es-For example: <i><a href="/subjects/person:Theodore_Roosevelt">Theodore Roosevelt</a>, <a href="/subjects/person:Julian_of_Norwich">Julian of Norwich</a>, <a href="/subjects/person:Tintin">Tintin</a>] - xml.etree.ElementTree.ParseError: mismatched tag: line 1, column 249
FAILED openlibrary/i18n/test_po_files.py::test_html_format[es-For example: <i><a href="/subjects/place:London">London</a>, <a href="/subjects/place:Atlantis">Atlantis</a>, <a href="/subjects/place:Omaha">Omaha</a>] - xml.etree.ElementTree.ParseError: mismatched tag: line 1, column 202
FAILED openlibrary/i18n/test_po_files.py::test_html_format[es-For example: <i><a href="/subjects/time:1984">1984</a>, <a href="/subjects/time:the_middle_ages">The Middle Ages</a>, <a href="/subjects/time:1810-1890">1810-1890</a>] - xml.etree.ElementTree.ParseError: mismatched tag: line 1, column 217
```

This strings should be full HTML. E.g. Include the opening `<a>` tag if missing, including the closing `</i>` tag, etc. Or in that Continue case, don't include the `</a>` at all.

### Reproducing the bug

1. Go to ...
2. Do ...

* Expected behavior:
* Actual behavior:


### Context

- Browser (Chrome, Safari, Firefox, etc):
- OS (Windows, Mac, etc):
- Logged in (Y/N): Y
- Environment (prod, dev, local): prod


### Notes from this Issue's Lead

#### Proposal & constraints
<!-- What is the proposed solution / implementation? Is there a precedent of this approach succeeding elsewhere? -->

#### Related files
<!-- Files related to this issue; this is super useful for new contributors who might want to help! If you're not sure, leave this blank; a maintainer will add them. -->

#### Stakeholders

@merwhite11

--- a/openlibrary/i18n/messages.pot
+++ b/openlibrary/i18n/messages.pot
@@ -1511,7 +1511,7 @@ msgid ""
 "query-sample\" href=\"\">sample</a> of the query used. <br />Improve "
 "these stats by adding the Wikidata author identifier following <a "
 "href=\"https://openlibrary.org/help/faq/editing.en#author-identifiers-"
-"purpose\"> these instructions</a>."
+"purpose\">these instructions</a>."
 msgstr ""
 
 #: account/readinglog_stats.html

--- a/openlibrary/i18n/messages.pot
+++ b/openlibrary/i18n/messages.pot
@@ -1508,8 +1508,8 @@ msgstr ""
 msgid ""
 "Demographic statistics powered by <a "
 "href=\"https://www.wikidata.org/\">Wikidata</a>. Here's a <a id=\"wd-"
-"query-sample\" href=\"\">sample</a> of the query used. <br>Improve these "
-"stats by adding the Wikidata author identifier following <a "
+"query-sample\" href=\"\">sample</a> of the query used. <br />Improve "
+"these stats by adding the Wikidata author identifier following <a "
 "href=\"https://openlibrary.org/help/faq/editing.en#author-identifiers-"
 "purpose\"> these instructions</a>."
 msgstr ""
@@ -2596,7 +2596,7 @@ msgid "None of these match the book I want to add."
 msgstr ""
 
 #: books/check.html
-msgid "Continue</a>."
+msgid "Continue"
 msgstr ""
 
 #: NotInLibrary.html books/custom_carousel.html
@@ -2633,8 +2633,8 @@ msgstr ""
 #: books/daisy.html
 #, python-format
 msgid ""
-"<strong>Download the DAISY.zip</strong></a> for <a "
-"href=\"%(url)s\">%(title)s</a>"
+"<a href=\"%(daisy_url)s\"><strong>Download the DAISY.zip</strong></a> for"
+" <a href=\"%(url)s\">%(title)s</a>"
 msgstr ""
 
 #: books/daisy.html
@@ -2879,7 +2879,7 @@ msgstr ""
 msgid ""
 "For example: <i><a href=\"/subjects/cheese\">cheese</a>, <a "
 "href=\"/subjects/roman_empire\">Roman Empire</a>, <a "
-"href=\"/subjects/psychology\">psychology</a>"
+"href=\"/subjects/psychology\">psychology</a></i>"
 msgstr ""
 
 #: books/edit/about.html
@@ -2890,7 +2890,7 @@ msgstr ""
 msgid ""
 "For example: <i><a href=\"/subjects/person:Theodore_Roosevelt\">Theodore "
 "Roosevelt</a>, <a href=\"/subjects/person:Julian_of_Norwich\">Julian of "
-"Norwich</a>, <a href=\"/subjects/person:Tintin\">Tintin</a>"
+"Norwich</a>, <a href=\"/subjects/person:Tintin\">Tintin</a></i>"
 msgstr ""
 
 #: books/edit/about.html
@@ -2901,7 +2901,7 @@ msgstr ""
 msgid ""
 "For example: <i><a href=\"/subjects/place:London\">London</a>, <a "
 "href=\"/subjects/place:Atlantis\">Atlantis</a>, <a "
-"href=\"/subjects/place:Omaha\">Omaha</a>"
+"href=\"/subjects/place:Omaha\">Omaha</a></i>"
 msgstr ""
 
 #: books/edit/about.html
@@ -2912,7 +2912,7 @@ msgstr ""
 msgid ""
 "For example: <i><a href=\"/subjects/time:1984\">1984</a>, <a "
 "href=\"/subjects/time:the_middle_ages\">The Middle Ages</a>, <a "
-"href=\"/subjects/time:1810-1890\">1810-1890</a>"
+"href=\"/subjects/time:1810-1890\">1810-1890</a></i>"
 msgstr ""
 
 #: books/edit/edition.html

--- a/openlibrary/templates/account/readinglog_stats.html
+++ b/openlibrary/templates/account/readinglog_stats.html
@@ -127,7 +127,7 @@ $jsdef render_excluded_works_list(works, total):
             </section>
     </section>
 
-    <small>$:_('Demographic statistics powered by <a href="https://www.wikidata.org/">Wikidata</a>. Here\'s a <a id="wd-query-sample" href="">sample</a> of the query used. <br />Improve these stats by adding the Wikidata author identifier following <a href="https://openlibrary.org/help/faq/editing.en#author-identifiers-purpose"> these instructions</a>.')</small>
+    <small>$:_('Demographic statistics powered by <a href="https://www.wikidata.org/">Wikidata</a>. Here\'s a <a id="wd-query-sample" href="">sample</a> of the query used. <br />Improve these stats by adding the Wikidata author identifier following <a href="https://openlibrary.org/help/faq/editing.en#author-identifiers-purpose">these instructions</a>.')</small>
 
     <h2>$_("Work Stats")</h2>
 

--- a/openlibrary/templates/account/readinglog_stats.html
+++ b/openlibrary/templates/account/readinglog_stats.html
@@ -127,7 +127,7 @@ $jsdef render_excluded_works_list(works, total):
             </section>
     </section>
 
-    <small>$:_('Demographic statistics powered by <a href="https://www.wikidata.org/">Wikidata</a>. Here\'s a <a id="wd-query-sample" href="">sample</a> of the query used. <br>Improve these stats by adding the Wikidata author identifier following <a href="https://openlibrary.org/help/faq/editing.en#author-identifiers-purpose"> these instructions</a>.')</small>
+    <small>$:_('Demographic statistics powered by <a href="https://www.wikidata.org/">Wikidata</a>. Here\'s a <a id="wd-query-sample" href="">sample</a> of the query used. <br />Improve these stats by adding the Wikidata author identifier following <a href="https://openlibrary.org/help/faq/editing.en#author-identifiers-purpose"> these instructions</a>.')</small>
 
     <h2>$_("Work Stats")</h2>
 

--- a/openlibrary/templates/books/check.html
+++ b/openlibrary/templates/books/check.html
@@ -39,7 +39,7 @@ $var title: $_("Add a book")
         </ul>
     </div>
 
-    <p class="small sansserif clearfix">$_("None of these match the book I want to add.") <a href="javascript:;" id="bookAddCont">$:_('Continue</a>.')</p>
+    <p class="small sansserif clearfix">$_("None of these match the book I want to add.") <a href="javascript:;" id="bookAddCont">$_('Continue')</a></p>
 
     <form method="POST" id="addbook" class="olform addbook1 hidden" name="edit">
         <input type="hidden" id="work" name="work" value="none-of-these"/>

--- a/openlibrary/templates/books/daisy.html
+++ b/openlibrary/templates/books/daisy.html
@@ -25,11 +25,11 @@ $ daisy = "//archive.org/download/XXX/XXX_daisy.zip"
             <p>$_("It can only be opened on a specialized device with a key issued by the Library of Congress.")</p>
         </div>
         <ul class="link">
-            <li><a href="$daisy.replace('XXX', page.ocaid)">$:_('<strong>Download the DAISY.zip</strong></a> for <a href="%(url)s">%(title)s</a>', url=page.key, title=websafe(page.title))</li>
+            <li>$:_('<a href="%(daisy_url)s"><strong>Download the DAISY.zip</strong></a> for <a href="%(url)s">%(title)s</a>', daisy_url=daisy.replace('XXX', page.ocaid), url=page.key, title=websafe(page.title))</li>
         </ul>
     $else:
         <ul class="link">
-            <li><a href="$daisy.replace('XXX', page.ocaid)">$:_('<strong>Download the DAISY.zip</strong></a> for <a href="%(url)s">%(title)s</a>', url=page.key, title=websafe(page.title))</li>
+            <li>$:_('<a href="%(daisy_url)s"><strong>Download the DAISY.zip</strong></a> for <a href="%(url)s">%(title)s</a>', daisy_url=daisy.replace('XXX', page.ocaid), url=page.key, title=websafe(page.title))</li>
         </ul>
 
 

--- a/openlibrary/templates/books/edit/about.html
+++ b/openlibrary/templates/books/edit/about.html
@@ -31,7 +31,7 @@ $jsdef render_subject_autocomplete_item(item):
     <div class="formElement">
         <div class="label">
             <label for="about-subjects">$_("Subject keywords?")</label>
-            <span class="tip"><span class="highlight">$_("Please separate with commas.")</span> $:_('For example: <i><a href="/subjects/cheese">cheese</a>, <a href="/subjects/roman_empire">Roman Empire</a>, <a href="/subjects/psychology">psychology</a>')</i></span>
+            <span class="tip"><span class="highlight">$_("Please separate with commas.")</span> $:_('For example: <i><a href="/subjects/cheese">cheese</a>, <a href="/subjects/roman_empire">Roman Empire</a>, <a href="/subjects/psychology">psychology</a></i>')</span>
         </div>
         $ config = {'name': 'subjects', 'facet': 'subject', 'data': work.get_subjects() if len(work.get_subjects()) else []}
         <div class="csv-autocomplete--subjects" data-config="$dumps(config)">
@@ -42,7 +42,7 @@ $jsdef render_subject_autocomplete_item(item):
     <div class="formElement">
         <div class="label">
             <label for="about-subject_people">$_("A person? Or, people?")</label>
-            <span class="tip">$:_('For example: <i><a href="/subjects/person:Theodore_Roosevelt">Theodore Roosevelt</a>, <a href="/subjects/person:Julian_of_Norwich">Julian of Norwich</a>, <a href="/subjects/person:Tintin">Tintin</a>')</i></span>
+            <span class="tip">$:_('For example: <i><a href="/subjects/person:Theodore_Roosevelt">Theodore Roosevelt</a>, <a href="/subjects/person:Julian_of_Norwich">Julian of Norwich</a>, <a href="/subjects/person:Tintin">Tintin</a></i>')</span>
         </div>
         $ config = {'name': 'subject_people', 'facet': 'person', 'data': work.subject_people if len(work.subject_people) else []}
         <div class="csv-autocomplete--subjects" data-config="$dumps(config)">
@@ -53,7 +53,7 @@ $jsdef render_subject_autocomplete_item(item):
     <div class="formElement">
         <div class="label">
             <label for="about-subject_places">$_("Note any places mentioned?")</label>
-            <span class="tip">$:_('For example: <i><a href="/subjects/place:London">London</a>, <a href="/subjects/place:Atlantis">Atlantis</a>, <a href="/subjects/place:Omaha">Omaha</a>')</i></span>
+            <span class="tip">$:_('For example: <i><a href="/subjects/place:London">London</a>, <a href="/subjects/place:Atlantis">Atlantis</a>, <a href="/subjects/place:Omaha">Omaha</a></i>')</span>
         </div>
         $ config = {'name': 'subject_places', 'facet': 'place', 'data': work.subject_places if len(work.subject_places) else []}
         <div class="csv-autocomplete--subjects" data-config="$dumps(config)">
@@ -64,7 +64,7 @@ $jsdef render_subject_autocomplete_item(item):
     <div class="formElement bottom">
         <div class="label">
             <label for="about-subject_times">$_("When is it set or about?")</label>
-            <span class="tip">$:_('For example: <i><a href="/subjects/time:1984">1984</a>, <a href="/subjects/time:the_middle_ages">The Middle Ages</a>, <a href="/subjects/time:1810-1890">1810-1890</a>')</i></span>
+            <span class="tip">$:_('For example: <i><a href="/subjects/time:1984">1984</a>, <a href="/subjects/time:the_middle_ages">The Middle Ages</a>, <a href="/subjects/time:1810-1890">1810-1890</a></i>')</span>
         </div>
         $ config = {'name': 'subject_times', 'facet': 'time', 'data': work.subject_times if len(work.subject_times) else []}
         <div class="csv-autocomplete--subjects" data-config="$dumps(config)">

